src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -61,7 +63,7 @@ the MIT License. See LICENSE in the project root for license information. -->
 
             <!--Disable Animations-->
             <local:SettingContainer x:Uid="Globals_DisableAnimations">
-                <ToggleSwitch IsOn="{x:Bind State.Globals.DisableAnimations, Mode=TwoWay}"/>
+                <ToggleSwitch IsOn="{x:Bind State.Globals.DisableAnimations, Mode=TwoWay, Converter={StaticResource InvertedBooleanConverter}}"/>

```

Comment:
I disagree fundamentally with this change. I'm not blocking you over it, but I don't think human interface guidebooks are bibles that we should sleep with under our pillows.

Reply:
_Technically_, the problem is that we should've named this setting something else. But that's much harder to fix (or we can do some kind of enum design, but I think _nobody_ wants that hahaha).

I do get what you're saying though. I think we just need to be more careful with these kinds of settings in the future. :/

Reply:
I really do not think we need to be more careful. It's the settings UI's job to properly represent whatever the underlying data model stores in whatever way is best for the user. "Disabling" things makes sense in the settings.

It's going to be far more annoying when we talk about "disabled" profile sources.

Reply:
Here's a setting I found in an app on my phone the other day:
![example](https://user-images.githubusercontent.com/11050425/108110244-d85e9c00-7047-11eb-80cd-2a2d4f96f1fd.png)
I didn't go out looking for this haha. I just ran into it and am now eternally confused by what that switch does. Btw, the gray text does _not_ update when you switch the value.

It's a real issue. We do need to be more careful.

Disabling profile sources is an issue for a separate discussion.

Reply:
I consider this switch to be on par with iOS's "Reduce motion" switch. It's on: motion is reduced. It is a switch that disables a thing when it is on.

Reply:
Chiming in here, I don't believe this is on par with "Reduce motion", since the switch is "enabling the reduction of motion" rather than "enabling the disabling of motion".

I think it makes sense to make the change to "Pane animations" and to do it now rather than wait for this to become the default settings experience.

Reply:
I mean, if we changed the wording to "reduce motion" it would 1. mean exactly the same thing and 2. do exactly the same thing (reduce motion), so... maybe we should do that? :smile:

Reply:
My concern with "reduce motion" is that there's an arbitrary toggle switch on the Appearance page that just says "reduce motion" without much context.

I'm trying to think of other wording that describes what happens when pane animations are disabled, that would help prevent switching the toggle logic. Something like "snap open panes"..?